lib/omniauth/auth0/jwt_validator.rb

```diff
@@ -15,7 +15,11 @@ class JWTValidator
       #   options.issuer - Application issuer (optional).
       #   options.client_id - Application Client ID.
       #   options.client_secret - Application Client Secret.
-      def initialize(options)
+      #   authorize_params[:leeway] - Clock-skew used for calculating time comparisons
+      #   authorize_params[:max_age] - Maximum amount of time a token is valid
```

Comment:
Are these clock params in seconds? It appears so, maybe should be explicit about that in the docs here, along with any defaults if no value is passed.